components/AddPotholeLocation.js

```diff
@@ -56,8 +56,14 @@ class AddPotholeLocation extends React.Component {
       longitudeDelta: 0.001,
     };
     this.setState({ initialRegion });
+    this._getPotholesAsync(latitude, longitude)
   };
 
+  _getPotholesAsync = async (lat, lon) => {
+    await this.props.getPotholes(lat, lon)
+    this.setState({potholes: this.props.potholes})
```

Comment:
is it necessary to map these to a local state? Did the reference to this.props.potholes not work in the render?